hstream-gossip/src/HStream/Gossip/Types.hs

```diff
@@ -60,8 +60,8 @@ defaultGossipOpts :: GossipOpts
 defaultGossipOpts = GossipOpts
   { gossipFanout          = 3
   , retransmitMult        = 4
-  , gossipInterval        = 1 * 1000 * 1000
-  , probeInterval         = 2 * 1000 * 1000
+  , gossipInterval        = 500 * 1000
+  , probeInterval         = 1 * 1000 * 1000
```

Comment:
If you update these default options, you may also want to update the config file under the conf dir(https://github.com/hstreamdb/hstream/blob/main/conf/hstream.yaml#L91)